icons/file_type_dvc.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill="#13adc7" d="M2.5 15.195v-8.54a.237.237 0 01.071-.174.222.222 0 01.161-.071H5.88c1.247 0 2.314.44 3.2 1.322.885.882 1.328 1.942 1.328 3.18 0 1.247-.443 2.314-1.328 3.2-.886.885-1.953 1.328-3.2 1.328H2.732a.222.222 0 01-.161-.07.237.237 0 01-.07-.175zm2.013-1.664H5.75c.723 0 1.322-.252 1.8-.755.477-.503.716-1.124.716-1.864 0-.731-.239-1.348-.716-1.851-.478-.503-1.077-.755-1.8-.755H4.513z"/><path fill="#945dd6" d="M11.703 17.577L7.678 8.895c-.043-.086-.04-.163.007-.232s.118-.103.212-.103h1.716c.112 0 .185.047.22.142l2.128 4.709h.078l2.128-4.71c.035-.094.108-.141.22-.141h1.715c.095 0 .166.034.213.103.048.069.05.146.007.232l-4.051 8.682c-.052.095-.125.142-.22.142h-.128c-.095 0-.168-.047-.22-.142z"/><path fill="#f46737" d="M14.969 14.228c-.903-.904-1.355-2-1.355-3.29s.454-2.389 1.361-3.296c.908-.908 2.006-1.361 3.296-1.361 1.205 0 2.245.4 3.122 1.2.13.12.134.24.013.36L20.4 8.888c-.112.103-.22.103-.322 0a2.526 2.526 0 00-1.742-.658c-.723 0-1.325.26-1.806.78-.482.52-.723 1.146-.723 1.877 0 .723.243 1.34.73 1.852.485.511 1.09.767 1.812.767.67 0 1.247-.206 1.729-.62.12-.102.232-.098.335.014l1.006 1.07c.112.112.108.228-.013.349-.86.834-1.905 1.251-3.135 1.251-1.29 0-2.39-.447-3.302-1.341z"/></svg>
```

Comment:
This requires an extra work as this doesn't match the export settings as stated on [wiki contributing](https://github.com/vscode-icons/vscode-icons/wiki/Contributing) section.